manabi/lock.py

```diff
@@ -150,18 +154,15 @@ class ManabiPostgresLock(ManabiContextLockMixin):
     _storage_object: Callable[[], "ManabiDbLockStorage"]
 
     def __init__(self, storage_object: "ManabiDbLockStorage"):
-        self._id = None
         # type manually checked
         self._storage_object = weakref.ref(storage_object)  # type: ignore
         self.acquire_write = self.acquire_read = self.acquire
         self._semaphore = 0
-        self._id = -1
+        super().__init__()
 
     def acquire(self):
         tid = threading.get_ident()
```

Comment:
Do we still need the get_ident here if we do it in the check_and_set_tid method?

Reply:
Yes, it's used in a log message a few lines below

Reply:
ahh thanks, didn't see that